<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CAEEFE;
background-color: rgb(255, 255, 255);
color:#000000</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../télécharger.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Quiter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 204, 153);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../nejma.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../télécharger.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Modifier_produit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;
</string>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>691</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CAEEFE;
font: 14pt &quot;Tempus Sans ITC&quot;;
color:#000000</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 204, 153);
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tempus Sans ITC'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; color:#0055ff;&quot;&gt;Gestion des données&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>241</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>411</width>
        <height>241</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(187, 240, 250);</string>
      </property>
      <property name="title">
       <string>Gestion de produit</string>
      </property>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;ID Produit :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;Nom produit :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;Prix :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;Quantité en stock :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;Catégorie :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;Date d'ajout :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_IDP">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_NomP">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prixP">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_quentit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dateP">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_quentit_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>41</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_produit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/p_m.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="num_pm">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>250</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>411</width>
        <height>241</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(187, 240, 250);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextEdit" name="textEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(153, 204, 153);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tempus Sans ITC'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:italic; text-decoration: underline;&quot;&gt;Autre matiére&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(187, 240, 250);
background-color: rgb(187, 240, 250);</string>
       </property>
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_51">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../logo-removebg-preview.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_52">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>49</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../nejma.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_53">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../dar.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>190</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../nejma.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../nejma.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableViewHistorique">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>281</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Historique d'ajout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLineEdit" name="find">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercherProduit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#99CC99</string>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>571</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../télécharger.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_58">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 204, 153);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../nejma.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../télécharger.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../dar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exporterEnPDF">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
    </property>
    <property name="text">
     <string>exportez en pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trier_produit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
    </property>
    <property name="text">
     <string>tri</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;</string>
    </property>
    <property name="text">
     <string>export arduino</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>121</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #CAEEFE; /* Fond bleu clair pour le groupe */
</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/images/Logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;
border-radius: 25px; 
border: 1.5px solid #29ABE2; 
color:#000000;
QPushButton: {
                     font-size: 14px;
                      transition: all 0.3s ease;
                      }
QPushButton:hover {
                      font-size: 18px;
                      }</string>
     </property>
     <property name="text">
      <string>produit</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
